src/emucore/Settings.cxx

```diff
@@ -27,6 +27,8 @@
 #include "bspf.hxx"
 #include "Settings.hxx"
 
+using namespace std;
```

Comment:
I would discourage a using directive for a single class (ostringstream), but this is fine as is.